hunting-buddy.lic

```diff
@@ -49,16 +49,32 @@ class HuntingBuddy
       end
 
       args = info['args']
+      before_actions = info['before']
+      after_actions = info['after']
       duration = info[:duration]
       stop_on_skills = info['stop_on']
 
+      #Execute scripts to run before the hunt
+      before_actions.each do |before_action|
+        action_parts = before_action.split(" ")
+        script_name = action_parts.shift
+        wait_for_script_to_complete(script_name, action_parts)
+      end
+
       next if all_skills_at_cap?(stop_on_skills)
```

Comment:
This line (and line 65) are two cases in which after actions would not occur. Is that ok, or will that cause problems?